### DISCUSSION TITLE
如何让Picker呈现 日-月-年 的选择视窗？

### DISCUSSION THREAD

**Discussion by eejoygp** [ant-design/ant-design-mobile#5444] [Q&A]
这是现在DatePicker能够做到的 可是我想要的是 日从左边开始到年
![Screenshot 2022-07-05 170151](https://user-images.githubusercontent.com/107380914/177326055-f7060d8f-5cf2-46a8-8cbc-4e244c8b2a9d.png)

**Comment by GOWxx** [comment]
对不起，我之前理解错了，错误的关闭了你的 issues.
我重新打开了它，抱歉 🙏

link https://github.com/ant-design/ant-design-mobile/issues/5441
